meta-balena-common/recipes-support/balena-config-vars/balena-config-vars/balena-config-vars

```diff
@@ -38,6 +38,28 @@ done
 BOOT_MOUNTPOINT="/mnt/boot"
 FLASHER_FILEFLAG="$BOOT_MOUNTPOINT/balena-image-flasher"
 
+fatcp() {
+    /usr/bin/fatrw copy "${1}" "${2}"
+}
+
+fatmv() {
+    fatcp "${1}" "${2}"
+    rm -rf "${1}"
+}
+
```

Comment:
Is it possible that the `rm` operation may leave the source file in a bad state on power cycle? Also do we need `fatmv` at all if we use do not use temporary files like I explained in another comment? If we need the `fatmv` indeed we may consider adding a sync after the `rm` (in fatrw we fsync the parent directory when we remove files).

Reply:
There are still some cases where using `fatmv` simplifies code. I will add the `sync` at the end though.